Fix : Enhance Image Viewer Usability with Standard Navigation

Jeffrin2005 opened the pull request:
**Description**

* Fixes  #6378

## Merge Checklist

- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.


https://github.com/user-attachments/assets/3b4fe2c3-1fbc-4c52-a480-4658104b5fe6


- [x] Completion of QA in Mobile Devices
- [x] Completion of QA in Desktop Devices



**Notes for Reviewers**


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit 5d7320b2af36b5e3c9a62a13d997339fba411590 at: https://67f6518833bdfdce99cfc21d--layer5.netlify.app

vr-varad commented:
@Jeffrin2005 Please don't change the formatting, rest everything looks good

Jeffrin2005 commented:
@vr-varad Formating means - the alignment of codes , right ?

l5io commented:
🚀 Preview for commit 12009941c99eb7bbdb6545c815cd6e82470a2d87 at: https://67f66c032056d6149b0ead0a--layer5.netlify.app

vr-varad commented:
LGTM @Jeffrin2005 
@vishalvivekm @amitamrutiya

vishalvivekm commented:
@Jeffrin2005 
Thank you for your contribution!
Let's discuss this during the website call today at 5:30 PM IST

Add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk), if you would :)

l5io commented:
🚀 Preview for commit 513d6ca6cf4565def838801cac666044e92db762 at: https://67fccb30c95f4abf8ace2400--layer5.netlify.app

vishalvivekm commented:
![Screen Shot 2025-04-14 at 16 20 35](https://github.com/user-attachments/assets/ac7f7e8a-c92d-453d-b0c3-985b15098cd0)

l5io commented:
🚀 Preview for commit 3910ba603e157f9bc515d3588465bb4b223122f4 at: https://67ff23f402d715b1102d71fb--layer5.netlify.app

Jeffrin2005 commented:
@vishalvi

https://github.com/user-attachments/assets/55884d65-07f0-4ec6-848e-b64c57ca25fe

vekm 

Now removed the box (behind the arrows) , and also fixed the responsive issue. 
Can you review this .

l5io commented:
🚀 Preview for commit 90bb7217cb201828175f4354e3b8ba7bf8ebfb26 at: https://680133a22c08f712c7a8eb14--layer5.netlify.app

Jeffrin2005 commented:
@vishalvivekm 

https://github.com/user-attachments/assets/55884d65-07f0-4ec6-848e-b64c57ca25fe



Now removed the box (behind the arrows) , and also fixed the responsive issue. 
Can you review this .

Jeffrin2005 commented:
@vishalvivekm 

https://github.com/user-attachments/assets/55884d65-07f0-4ec6-848e-b64c57ca25fe



Now removed the box (behind the arrows) , and also fixed the responsive issue. 
Can you review this .

Jeffrin2005 commented:
@vishalvivekm @leecalcote Can you review this PR?

l5io commented:
🚀 Preview for commit 5818ce05c915744368b0af7c9af392561c204e6f at: https://685fbcdaf9c1d68479d769fe--layer5.netlify.app
